src/pages/account/security/update.tsx

```diff
@@ -75,48 +70,10 @@ const getServerSideProps: GetServerSideProps = async (context) => {
       errorMiddleware(getTracer()),
       loggingMiddleware(getTracer(), logger),
       authMiddleware(getTracer()),
+      verifySudoMode(getTracer(), context.resolvedUrl),
     ],
     "/account/security/update",
-    async (request) => {
-      const webErrors = {
-        databaseUnreachable: ERRORS_DATA.DATABASE_UNREACHABLE,
-        noUserFound: ERRORS_DATA.NO_USER_FOUND,
-        sudoModeTTLNotFound: ERRORS_DATA.SUDO_MODE_TTL_NOT_FOUND,
-      };
-
-      const userCookie = await getServerSideCookies<UserCookie>(request, {
-        cookieName: "user-cookie",
-        isCookieSealed: true,
-        cookieSalt: configVariables.cookieSalt,
-      });
-
-      if (!userCookie) {
-        throw new NoUserCookieFoundError();
-      }
-
-      const user = await getDBUserFromSub(userCookie.sub).catch((error) => {
-        throw webErrorFactory({
-          ...webErrors.databaseUnreachable,
-          parentError: error,
-        });
-      });
-
-      if (!user) {
-        throw webErrorFactory(webErrors.noUserFound);
-      }
-
-      const sudoModeTTL = user.sudo.sudo_mode_ttl;
-
-      if (!sudoModeTTL || Date.now() > sudoModeTTL) {
-        const urlBeforeRedirect = context.resolvedUrl;
-        return {
-          redirect: {
-            destination: `/account/security/sudo?next=${urlBeforeRedirect}`,
-            permanent: false,
-          },
-        };
-      }
-
+    async () => {
       return {
         props: {},
       };
```

Comment:
You can remove this. If no handler are given as argument, it will default to
```
() => {
      return {
        props: {},
      };
```

Also, this is not an async function :).